src/main/plugins/org.dita.base/build_preprocess2_template.xml

```diff
@@ -354,6 +354,7 @@ See the accompanying LICENSE file for applicable license.
           <param name="processing-mode" value="${processing-mode}" if:set="processing-mode"/>
         </filter>
         <filter class="org.dita.dost.writer.CoderefResolver" unless:set="preprocess.coderef.skip"/>
+        <filter class="org.dita.dost.writer.NormalizeCodeblock" unless:set="preprocess.codeblock.skip"/>
```

Comment:
Is this a good guard condition property name? Maybe `preprocess.normalize-codeblock.skip` would be better?

Reply:
I think `preprocess.normalize-codeblock.skip` is probably better? It feels too long -- but like many of the `skip` conditions, it will probably only be seen by / used by a small number of people, so length is not a big issue.

Reply:
Changed